.github/workflows/run-tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       fail-fast: true
       matrix:
         os: [ubuntu-latest, windows-latest]
-        php: [8.0, 8.1]
+        php: [8.2]
```

Comment:
Please make it compatible again with PHP 8.0 and above. I think you'd only need to drop that `readonly` keyword.